broker/src/test/java/win/liyufan/im/UtilityTest.java

```diff
@@ -0,0 +1,14 @@
+package win.liyufan.im;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class UtilityTest {
+    @Test
+    public void formatJsonInput() {
+        Assert.assertEquals("", Utility.formatJson(null));
+        Assert.assertEquals("1", Utility.formatJson("1"));
+        // Hand-written, as inspired by generated tests
```

Comment:
Maybe remove this comment. We don't tend to use the word `generated`.
Although the tests were initially created by Diffblue Cover they have been subsequently tailored to the repo so I don't think it's worth highlighting a difference between the tests' origins.

Reply:
Ok. Removed.